packages/apps/plugins/plugin-space/src/components/SpacePresence.tsx

```diff
@@ -28,6 +28,9 @@ import { ComplexMap, keyToFallback } from '@dxos/util';
 import { SPACE_PLUGIN } from '../meta';
 import type { ObjectViewerProps, SpacePluginProvides } from '../types';
 
+const REFRESH_INTERVAL = 10_000;
+const ACTIVITY_DURATION = 30_000;
```

Comment:
ideally we could align this with the constant that the plugin uses to set the interval for broadcasting updates

Reply:
Is one available?